dougP/helloworld.js

```diff
@@ -0,0 +1 @@
+console.log("hello, dougP");
```

Comment:
Great Work!

in the future when you name your directory you should use a lower case dougp
capital-letters can cause problems with different types of file systems, when you work in groups :)

moste people use _ to seporate words in file names, some people use -